examples/demo.py

```diff
@@ -0,0 +1,25 @@
+# -------------------------------------------------------------------------
+#
+#  Copyright (c) 2005, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in enthought/LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+#  Author: David C. Morrill
+#  Date:   09/15/2005
+#
+# -------------------------------------------------------------------------
```

Comment:
Brought this from traitsui, should this be modified?

Reply:
It should. The current copyright header policy for Enthought OSS projects is - 

```
# (C) Copyright 20XX-20XX Enthought, Inc., Austin, TX
# All rights reserved.
```